recipe/meta.yaml

```diff
@@ -22,19 +22,22 @@ requirements:
     - python
   run:
     - fastcache
-    - setuptools
     - mpmath >=0.19
     - python
     - gmpy2 >=2.0.8   # [not win]
+  run_constrained:
+    - antlr-python-runtime >=4.7,<4.8  # [not aarch64 and not ppc64le]
 
 test:
+  requires:
+    - antlr-python-runtime
```

Comment:
Maybe this line also needs to be skipped?

Reply:
yes, indeed!